changelog/8.11.1.asciidoc

```diff
@@ -0,0 +1,33 @@
+// begin 8.11.1 relnotes
+
+[[release-notes-8.11.1]]
+==  8.11.1
+
+Review important information about the  8.11.1 release.
```

Comment:
We need to document that the known ECE issue in 8.11.0 is fixed:

https://github.com/elastic/ingest-docs/blob/ca3a091f2e7c760626dd04a6e675ce2c2dd23879/docs/en/ingest-management/release-notes/release-notes-8.11.asciidoc?plain=1#L49-L58

Done by https://github.com/elastic/elastic-agent/commit/5cd6efe7c2b69187012b7ecb027cbad2cc061224

Reply:
I will add a note about this to the 8.11.1 release notes when I create them today.